src/fluid-relay/azext_fluid_relay/azext_metadata.json

```
@@ -0,0 +1,4 @@
+{
+    "azext.isExperimental": true,
+    "azext.minCliCoreVersion": "2.38.0"
```

Should wait for the release of Azure CLI 2.39.0
```suggestion
    "azext.minCliCoreVersion": "2.39.0"
```

---

Thank you for your correction! Relevant questions have been revised.